sdl-core/src/main/scala/io/smartdatalake/definitions/Environment.scala

```diff
@@ -50,90 +51,150 @@ object Environment {
    * The environment parameter can contain multiple authorities separated by comma.
    * An authority is compared against the filesystem URI with contains(...)
    */
-  var hadoopAuthoritiesWithAclsRequired: Seq[String] = {
-    EnvironmentUtil.getSdlParameter("hadoopAuthoritiesWithAclsRequired")
-      .toSeq.flatMap(_.split(','))
+  def hadoopAuthoritiesWithAclsRequired: Seq[String] = {
+    if (_hadoopAuthoritiesWithAclsRequired.isEmpty) {
+      _hadoopAuthoritiesWithAclsRequired = Some(
+        EnvironmentUtil.getSdlParameter("hadoopAuthoritiesWithAclsRequired")
+          .toSeq.flatMap(_.split(','))
+      )
+    }
+    _hadoopAuthoritiesWithAclsRequired.get
   }
+  var _hadoopAuthoritiesWithAclsRequired: Option[Seq[String]] = None
```

Comment:
What's the reason for not making this private? Only Unit Tests?
If yes, how about making it package private?
```suggestion
  private [smartdatalake] var _hadoopAuthoritiesWithAclsRequired: Option[Seq[String]] = None
```